src/backend/marsha/core/tests/test_command_send_reminders.py

```diff
@@ -1343,3 +1347,53 @@ def test_scenario_video_date_has_changed(self):
         self.assertEqual(livesession_2.must_notify, ["DATA"])
         self.assertEqual(livesession_2.reminders, ["ONE", key_updated])
         self.assertEqual(livesession.reminders, [key_updated, key_updated])
+
+    def test_send_reminders_date_i18n(self):
+        """Livesession's language is used to send email."""
+
+        video = VideoFactory(
+            live_state=IDLE,
+            live_type=RAW,
+            starting_at=timezone.now() + timedelta(days=200),
+        )
+        # LTI livesession
+        lti_livesession = LiveSessionFactory(
+            consumer_site=video.playlist.consumer_site,
+            created_on=timezone.now() - timedelta(days=32),
+            email="chantal@test-fun-mooc.fr",
+            is_registered=True,
+            language="fr",
+            lti_id="Maths",
+            lti_user_id="56255f3807599c377bf0e5bf072359fd",
+            must_notify=[settings.REMINDER_DATE_UPDATED],
+            should_send_reminders=True,
+            video=video,
+        )
+
+        out = StringIO()
+        call_command("send_reminders", stdout=out)
+        self.assertIn(
+            "Mail sent chantal@test-fun-mooc.fr Votre webinaire a été mis à jour.",
+            out.getvalue(),
+        )
+        out.close()
+        self.assertEqual(len(mail.outbox), 1)
+
+        # check we send it to the the right mail.outbox[0].to[0]
+        self.assertEqual("chantal@test-fun-mooc.fr", mail.outbox[0].to[0])
+        # check it's the right content
+
+        self.assertIn(
+            f"Accéder au webinaire [//example.com/videos/{lti_livesession.video.pk}?lrpk="
+            f"{lti_livesession.pk}&amp;key={lti_livesession.get_generate_salted_hmac()}]",
+            " ".join(mail.outbox[0].body.split()),
```

Comment:
as you are computing this several times, you can probably save it in the variable first and then use it everywhere you need ?